src/test/java/org/sonarsource/scanner/maven/bootstrap/ScannerBootstrapperTest.java

```diff
@@ -333,6 +328,19 @@ void maven_opts_is_not_logged_at_info_level_when_not_absent_from_environment_var
     }
   }
 
+  private void setSonarScannerScanAllTo(String value) {
+    Properties withScanAllSet = new Properties();
+    withScanAllSet.put(MavenScannerProperties.PROJECT_SCAN_ALL_SOURCES, value);
+    when(session.getUserProperties()).thenReturn(withScanAllSet);
+  }
+
+  private void verifyCollectedSources(Consumer<String[]> sourceDirsAssertions) throws MojoExecutionException {
```

Comment:
NIce idea with these methods, thanks for simplifying testing.

Reply:
Thank you :)